main.css

```diff
@@ -1,419 +0,0 @@
-}
-
-/*--------------------------------------------------------------*/
-/* Header Styles */
-
-#title{
-    text-shadow: 2px 2px black;
-    grid-area: title;
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    font-size: 45px;
-    margin-bottom: 13vh;
-    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
-    font-weight: 900;
-    color: lightblue;  
-}
-
-a{
-    text-decoration: none;
-}
-
-div[id^="dorp"]{
-    margin: 5px;
-    font-size: 15px;
-    border: solid 2px #84A5b8;
-    text-decoration: none;
-}
-
-div[id^="dorp"]:hover{
-    box-shadow: 2px 2px 2px black;
-}
-
-#drop-down{
-    margin-top: 60px;
-    position: absolute; 
-}
-
-#header:hover
-#drop-down{
-    display: inline-block;
-    animation: slide 1.5s;
-    animation-fill-mode: forwards;
-    margin-top: 60px;
-}
-
-/*-------------------------------------------------------*/
-/* For The Menu bar slide animation*/
-
-@-webkit-keyframes slide {
-    0%{ right: 50vw}
-    100% { right: 27vw; }
-}
-
-@keyframes slide {
-    0%{ right: 50vw}
-    100% { right: 27vw; }
-}
-
-#header{
-    margin-bottom: 2vh;
-    border-bottom: double lightblue 10px;
-    grid-area: header;
-    font-family: fantasy;
-    font-size: 50px;
-    height: auto;
-    text-align:center;
-    display: flex;
-    justify-content: center;
-}
-
-
-/*--------------------------------------------------------------*/
-/* Top Images Styles */
-
-#topImgHolder{
-    grid-area: topImgHolder;
-    height: auto;
-    display: flex;
-    justify-content: center;;
-    margin-bottom: 13vh;
-    
-}
-.firstImgs{
-    box-shadow: 2px 2px 2px;
-    border: 10px double lightblue;
-    max-width: 300px;
-    max-height: 200px;
-}
-.caption{
-    margin-top: 10px;
-    text-align:  center;
-    font-family:'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
-    font-style: italic;
-    font-size: 18px;
-}
-
-/*--------------------------------------------------------------*/
-/* Quote Bar Styles */
-
-#quoteBar{
-    background-image: url('imgs/1234.jpg');
-    margin-bottom: 10vh;
-    display: flex;
-    height: 150px;
-    grid-area: quoteBar;
-    align-items: center;
-    justify-content: center;
-}
-
-/*--------------------------------------------------------------*/
-/* Main Articles Styles */
-
-#main{
-    padding: 5px;
-    margin: 10px;
-    grid-area: main;
-    height: 100vh;
-    overflow: scroll;   
-}
-.continueReading{
-    font-size: 12px;
-    margin-bottom: 15px;
-    max-width: 100px;
-    height: 20px;
-    display: flex;
-    align-items: center;
-    justify-content: center;
-    background-color:rgb(112, 138, 153);  
-    box-shadow: 1px 1px 1px black;
-    color: white;
-}
-.continueReading:hover{
-    box-shadow: 2px 2px 2px black
-}
-.articleImgs{
-    max-width:35vw;
-    height: auto;
-    border: rgb(179, 218, 241) solid 2px;
-}
-.LearnMore{
-    font-size: 12px;
-    margin-bottom: 15px;
-    max-width: 100px;
-    height: 20px;
```

Comment:
To easily make your page more reesponsive use relative units such as `%`, `vw`, or `vh`